SignalProcessing/FIRFilter.cpp

```diff
@@ -7,7 +7,9 @@
 //
 
 #include "FIRFilter.hpp"
+
 #include <cmath>
+#include <cstddef>
```

Comment:
<cstddef> should be included **before** "FIRFilter.hpp" or moved in "FIRFilter.hpp".
Everything else works :)

Reply:
Thanks again! Corrected in #954